app/Http/Controllers/API/V1/ReparationRequestMaterialController.php

```diff
@@ -34,4 +36,15 @@ public function index(Request $request): LengthAwarePaginator
 
         return $materialRequest;
     }
+
+    public function store(ReparationRequestMaterialStoreRequest $reparationRequestMaterialStoreRequest): ReparationRequestMaterial
+    {
+        $reparationRequest = ReparationRequest::find((int) $reparationRequestMaterialStoreRequest->get('reparation_request_id'));
```

Comment:
A policy is missing for this model and ability. Since we don't have a concrete solution fore this, we can specify the permission check logic later.